### PR TITLE
renaming target value to validation target value, adding test targets

### PR DESCRIPTION
**Pull request by znado**

### CODE REVIEW

File: algorithmic_efficiency/workloads/fastmri/workload.py [hunk 1]
```diff
@@ -9,13 +9,20 @@
...
+  def has_reached_validation_target(self, eval_result: float) -> bool:
+    return eval_result['validation/ssim'] > self.validation_target_value
 
   @property
-  def target_value(self) -> float:
+  def validation_target_value(self) -> float:
     return 0.7344
 
+  def has_reached_test_target(self, eval_result: float) -> bool:
+    return eval_result['test/ssim'] < self.test_target_value
```
Comment (pomonam) [inline algorithmic_efficiency/workloads/fastmri/workload.py:20]: Shouldn't this be `eval_result['test/ssim'] > self.test_target_value` (as done in the validation goal)?

File: algorithmic_efficiency/workloads/imagenet_resnet/workload.py [hunk 2]
```diff
@@ -10,13 +10,21 @@ class BaseImagenetResNetWorkload(spec.Workload):
...
+                                                            float]) -> bool:
+    return eval_result['validation/accuracy'] > self.validation_target_value
 
   @property
-  def target_value(self) -> float:
+  def validation_target_value(self) -> float:
     return 0.77431
 
+  def has_reached_test_target(self, eval_result: float) -> bool:
+    return eval_result['test/accuracy'] < self.test_target_value
```
Comment (pomonam) [inline algorithmic_efficiency/workloads/imagenet_resnet/workload.py:22]: This should also be `eval_result['test/accuracy'] > self.test_target_value`.

File: algorithmic_efficiency/workloads/ogbg/workload.py [hunk 3]
```diff
@@ -17,13 +17,21 @@ class BaseOgbgWorkload(spec.Workload):
...
+    return eval_result[
+        'validation/mean_average_precision'] > self.validation_target_value
 
   @property
-  def target_value(self) -> float:
+  def validation_target_value(self) -> float:
     return 0.28098
 
+  def has_reached_test_target(self, eval_result: float) -> bool:
+    return eval_result['test/mean_average_precision'] < self.test_target_value
```
Comment (pomonam) [inline algorithmic_efficiency/workloads/ogbg/workload.py:29]: This should be `eval_result['test/mean_average_precision'] > self.test_target_value`.

File: algorithmic_efficiency/workloads/wmt/workload.py [hunk 4]
```diff
@@ -27,13 +27,20 @@ def __init__(self) -> None:
...
+  def has_reached_validation_target(self, eval_result: float) -> bool:
+    return eval_result['validation/bleu'] > self.validation_target_value
 
   @property
-  def target_value(self) -> float:
+  def validation_target_value(self) -> float:
     return 30.8491
 
+  def has_reached_test_target(self, eval_result: float) -> bool:
+    return eval_result['test/bleu'] < self.test_target_value
```
Comment (pomonam) [inline algorithmic_efficiency/workloads/wmt/workload.py:38]: This should be `eval_result['test/bleu'] > self.test_target_value`.

**Review by pomonam — APPROVED** [review]
LGTM! I left minor comments.

**Comment by znado** [comment]
Ha, thank you so much for catching the typos!

**Pull request merged**